variants/aws-dev/Cargo.toml

```diff
@@ -52,3 +52,4 @@ iputils = { path = "../../packages/iputils" }
 strace = { path = "../../packages/strace" }
 tcpdump = { path = "../../packages/tcpdump" }
 chrony = { path = "../../packages/chrony" }
+ecr-credential-provider = { path = "../../packages/ecr-credential-provider" }
```

Comment:
I think it should be safe to declare `ecr-credential-provider` as a dependency of each kubernetes package, the reason being that now the kubelet has `KubeletCredentialProviders` and it actually will depend on the binaries provided by this package.